src/gsCore/gsConstantFunction.h

```diff
@@ -16,6 +16,8 @@
 #include <gsCore/gsLinearAlgebra.h>
 #include <gsCore/gsGeometry.h>
 #include <gsUtils/gsCombinatorics.h>
+#include <gsIO/gsXml.h>
+#include <gsIO/gsXmlGenericUtils.hpp>
```

Comment:
we cannot include a .hpp in an .h file.
We need to add .hpp and _.cpp for this...